src/client/stylesheets/style.less

```diff
@@ -283,8 +283,15 @@ a.contact-text:visited {
 }
 
 .deleted {
+	text-decoration: @brand-danger line-through;
```

Comment:
This doesn't work everywhere I tested. On IE 11 and Opera 43 for example `text-decoration: line-through` works, but `text-decoration: $SOME_COLOR line-through;` doesn't.

We can add both one on top of each other; the browsers that don't recognize the color form will fallback automatically to the  form without the color.

Reply:
Tried a different approach this time, tested it in all the listed browser. Works on all of them :)
Also i have removed all the css vendor prefixes because **text-decoration** property is well supported in all the browsers.

Reply:
Great, that looks even simpler :) I'll look at it and try to deploy to test.bookbrainz.org so we can test IRL